### PR TITLE
Fixes #704: Update travis file

### LINKED ISSUE #704: Update travis to fix npm cache
**I'm submitting a ...**
- [X] bug report
- [ ] feature request

**Current behavior:**

Currently there is incorrect ordering of ```cache```, and cached directory in travis which is leading to error in travis build. 

**Expected behavior:**

travis build check for node v6.14.2 should not affect the check for v6.9.1

**Screenshots of the issue:**

Not applicable

### PR DESCRIPTION
**Pull request by simsausaurabh**
**Changes proposed in this pull request**
- Corrected order of ```cache``` parameter.

**Screenshots (if appropriate)** Not applicable

**Link to live demo: http://pr-705-fossasia-loklaksearch.surge.sh** 

**Closes #704**

### CODE REVIEW

**Comment by simsausaurabh** [comment]
@singhpratyush @jigyasa-grover @daminisatya @sudheesh001 please review.

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,13 @@ language: node_js
 node_js:
   - 6
 
+cache:
+  directories:
+    - $HOME/.cache/pip
```
Comment (singhpratyush) [inline .travis.yml]: Is this needed? I don't think `pip` is used in this project anywhere. Also, why is `node_modules` being removed from cache?

Comment (simsausaurabh) [reply .travis.yml]: Sorry, I have made a mistake here. This is not needed. Working on it.

**Comment by simsausaurabh** [comment]
Caching the node_modules folder is causing problems (build fails) while caching the .npm cache will avoid it. It's mainly because the .npm cache doesn't store compiled native modules while the node_modules cache does. So while testing different versions of Node, in our case v6.*(node v6.14.2 and v6.9.1 and others), it will try to load a native module compiled for v6.14.2 in v6.9.1. It is the cause of failing travis of PR - #672

**Pull request merged**